marketplace/connect/client.py

```diff
@@ -23,24 +23,21 @@ def auth_header(self) -> dict:
         return {"Authorization": self.__get_auth_token()}
 
 
-class ConnectProjectClient(ConnectAuth):
+class ConnectProjectClient(ConnectAuth): # TODO: change class name to FlowsRESTClient
```

Comment:
Need least two spaces before inline comment like:
`class ConnectProjectClient(ConnectAuth):  # TODO: change class name to FlowsRESTClient`